### PR TITLE
Collection of forward ports / updates for downstream projects

### PR DESCRIPTION
**Pull request by andrewkfiedler**
- Fixes an issue that cropped up in a downstream project.
 - Forward ports of various prs, see commits for breakdown.

Hero Instructions:
```
Build ddf 2.26.10
Start ddf
profile:install standard
feature:repo-add mvn:com.connexta.jsonrpc/jsonrpc-features/0.8-SNAPSHOT/xml/features
feature:install jsonrpc-2.25
feature:repo-add mvn:org.codice.ddf.search/intrigue-ui-app/5.0.1-SNAPSHOT/xml/features
feature:install catalog-ui-app
feature:repo-add mvn:org.codice.ddf.search/ui-frontend/5.0.1-SNAPSHOT/xml/features
feature:install ui-frontend
Now you're good to go
```

### CODE REVIEW

**Comment by andrewkfiedler** [comment]
build now

**Review by glenhein — APPROVED** [review]
I approved this PR, but there is so much going on in this PR that I really felt lost with what's really going on.

File: pom.xml [hunk 1]
```diff
@@ -56,7 +56,7 @@
     <errorprone.version>2.2.0</errorprone.version>
 
     <!--Backend properties-->
-    <ddf.version>2.26.5</ddf.version>
+    <ddf.version>2.26.10</ddf.version>
```
Comment (jlcsmith) [inline pom.xml:59]: can bump to 2.26.12

**Review by jlcsmith — APPROVED** [review]
mainly reviewed just the backend code but looks good to me

**Comment by jlcsmith** [comment]
🎉  Hero Success 🎉 
Went through the UI and looked at the various changes - exports, query federation, drop downs, advanced search, user settings, etc.  Only thing of note is that the theme settings page does not use the full vertical extent like the other settings.  This may have been existing 
image

**Comment by jlcsmith** [comment]
@andrewkfiedler pointed that the theme setting page is meant to allow users to see the theme changes - so works as expected.

**Pull request merged**